octoprint_mrbeam/templates/tab_workingarea.jinja2

```diff
@@ -51,8 +51,7 @@
 
 													<div class="row-fluid print-control btn-group" style="display: none;" data-bind="visible: loginState.isUser">
 														<button id="laser_button" data-test="working-area-laser-button" class="btn btn-danger span6"
-                                                                data-bind="click: conversion.show_conversion_dialog, enable: {{ 'true' if isDevelop and 'enableLaserButton' in request.args else 'false'}} || (state.isOperational() && state.isReady() && !state.isPrinting() && !state.isPaused() && loginState.isUser() && !workingArea.working_area_empty()), attr: {title: state.titlePrintButton}"
-                                                                id="job_print">
```

Comment:
What exactly is this ID? Were there 2 IDs for the same element?

Reply:
Yes there were two ids before thats why the whole guided tour didn't work before.
I seached for it and couldn'T find its usage, I came from the original octoprint print button.